routing/missioncontrol.go

```diff
@@ -342,6 +342,24 @@ func (m *MissionControl) GetHistorySnapshot() *MissionControlSnapshot {
 	return m.state.getSnapshot()
 }
 
+// ImportHistory imports the set of mission control results provided to our
+// in-memory state. These results are not persisted, so will not survive
+// restarts.
+func (m *MissionControl) ImportHistory(history *MissionControlSnapshot) error {
+	if history == nil {
+		return errors.New("cannot import nil history")
+	}
+
+	m.Lock()
+	defer m.Unlock()
+
+	log.Debugf("Importing history snapshot to mission control")
```

Comment:
nit: maybe. bump this to `Infof`? would also make a nice addition to log how many were added/ignored! that way info users only see:
```
[CRTR] Importing history snapshot to mission control
[CRTR] Imported XX/YY fresh results from mission control history snapshot.
```